lib/formtastic/helpers/input_helper.rb

```diff
@@ -283,7 +283,7 @@ def default_input_type(method, options = {}) # @private
             return :time_select
           when :date
             return :date_select
-          when :hstore
+          when :hstore, :json, :jsonb, :citext
             return :text
```

Comment:
text database type renders a text input, so it makes sense to me that citext would also render text.